docker-compose.yml

```diff
@@ -34,6 +34,21 @@ services:
     environment:
       ACCEPT_EULA: "Y"
       SA_PASSWORD: MyS3cr3tPassw0rd
+   
+   # Setup the runner
+  # we need to set the localhost and root password as they can be different
+  lighthouse:
+    build: .
+    environment:
+      - LOCALHOST=host.docker.internal
+      - FLASK_APP=lighthouse
+      - FLASK_ENV=development
+      - EVE_SETTINGS=development.py
+      - ROOT_PASSWORD=
```

Comment:
Where is this used?

Reply:
Without it nothing works for me. It uses them as environment variables. I couldn't get it to read the .env file.

Reply:
But I can't find a reference in the codebase for `ROOT_PASSWORD`?